modules/sdk/src/service/index.ts

```diff
@@ -41,6 +41,14 @@ import {
 
 export { BrowserNode, ERC20Abi, FullChannelState, getBalanceForAssetId, TransferQuote, VectorError };
 
+const LOGDNA_INGESTION_KEY = "86e2bf58c669c293e6525e49eab86a0c";
```

Comment:
Why don't we just import this from the config?

Reply:
done